server/api/unsafe_operation.go

```diff
@@ -48,24 +48,30 @@ func (h *unsafeOperationHandler) RemoveFailedStores(w http.ResponseWriter, r *ht
 	rc := getCluster(r)
 	var input map[string]interface{}
 	if err := apiutil.ReadJSONRespondError(h.rd, w, r.Body, &input); err != nil {
+		h.rd.JSON(w, http.StatusBadRequest, err)
 		return
 	}
-	storeSlice, ok := typeutil.JSONToUint64Slice(input["stores"])
-	if !ok {
-		h.rd.JSON(w, http.StatusBadRequest, "Store ids are invalid")
-		return
-	}
+
+	_, force := input["force"]
```

Comment:
How about using a query?

Reply:
what do you mean?

Reply:
Something like r.URL.Query().Get("force")